arches/app/templates/base-manager.htm

```diff
@@ -244,6 +245,25 @@ <h5>{% trans "for more documentation, visit" %} <a href="https://arches.readthed
                                         </li>
                                         {% endif %}
 
+                                        
+                                        <li>
```

Comment:
we need to use the `active-sub` class here when the user is on the search page, by checking the `main-script` like so:
```
{% if "views/search" in main_script %} class="active-sub"{% endif %}
```